data/hai/hai culture conversations.txt

```diff
@@ -329,3 +329,55 @@ mission "Hai Film Release"
+				has "language: Hai"
+			`	You freeze for a second, as it is unclear what the Hai wants from you.`
+				goto yourorder
+			label translator
+			`	Your translator chirps dutifully, "Have a [seat, rest] anywhere you like! I'll be right with you."`
+			label yourorder
+			`	Only a short moment later, the same Hai hostess peers over at you and chitters in what you assume is embarrassment. "I'm so sorry. I didn't realize I would have the pleasure of serving a human today. Please, have a seat anywhere at the bar. I will come to take your order shortly."`
+			`	You take a look around and note that the restaurant is actually quite crowded. There are Hai dressed in fancy tuxedos, fashionable clothing, casual dress, and even some wearing peculiar overalls with holes in the backs for their tails; there don't seem to be any other humans. With only a few seats available, you select one on the end, next to a rather fat Hai wearing green overalls and an orange straw hat. He grunts noncommittally as you sit down on a stool clearly designed for an organism with a tail. Before long, the hostess approaches and asks for your order.`
+			choice
+				`"I'll have today's special."`
+				`"I'd like some soup, please."`
+			`	"You're in luck," she says. "Today only, each meal is served with a free side of acorn bread." She scampers off to place and retrieve your order.`
+			`	The chubby Hai next to you grunts again - this time with clear frustration - and turns to speak. "Acorn bread! That's all they serve here anymore. Ever since you humans arrived in our space, that's all we get to eat. Traditional Hai breadmaking has become a lost art."`
+			choice
+				`"You mean there's more than just acorn bread?"`
+					goto breads
+				`"I'm sorry. I didn't mean to offend your culture."`
+			`	The look on his face softens. "I should be the one to apologize. It is not your fault that Hai culture has somewhat eroded in this respect. Humans have also brought new life to many parts of Hai society. It is only a pity that we lack the many types of bread that we had in years past."`
+			`	"You mean there was more than just acorn bread?" you ask.`
+			label breads
+			`	"Oh yes. Acorn bread was a rare sight in Hai cuisine. It's too sweet, you see." He shifts his weight on the stool. "I'm old enough to remember hayipp bread, yur ki bread - you name it. Hayipp is a grain, similar to your human 'corn,' which creates a wholesome ruddy brown flour when milled. The bread was hearty, yet exquisite." You note the wistful look on his face. "Yur ki is a milk, sourced from the livestock we raise. You may have seen some on Newhome; I'm told they resemble your human 'sheep.' Bread made from yur ki is rich and decadent. It is among my favorite foods to have for breakfast." He pauses for a moment and looks a bit downcast. "Since humans have joined us on our planets, all they want is acorn bread. The high demand has displaced all of the traditional breads that I grew up with. I miss them."`
```

Comment:
In line with the above comment - the expected difficulty would be that if you lived near a spaceport for work or business reasons, it might be annoyingly difficult to source more traditional breads.